src/datareaders/table_enumerations.py

```diff
@@ -0,0 +1,10 @@
+'''
+Should extend enum.  Hardcoded sources
```

Comment:
Whats the enum class that we want to extend here?

Reply:
I think that it was a common way to ensure that our sources stay consistent in their identifiers on both our db and in our code.  We don't need to extend anymore, but it might be good practice

Reply:
enum is a built in python module for enumerations (in this case, remembering which index/number matches which source name). It's working fine now, but we think that enum is how most people do this sort of thing. We couldn't extend it before for some reason (maybe the server doesn't have it installed). Regardless, it's not a very important thing since it works fine this way.

Reply:
Gotcha, just wanted to make sure I wasn't missing a class